framework/test/jest/integration/specs/modules/chain/scenarios/duplicate_signatures.spec.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+describe('duplicate_signatures', () => {
```

Comment:
The file name might be misleading as this also tests happy path so maybe it should be renamed to `multisignatures` or something similar?